action_plugins/load_profile/__init__.py

```diff
@@ -133,24 +132,27 @@ def __init__(
         super().__init__(behavior_type)
 
         # Model variables
-        self.profile_filename = ""
+        self._profileFilename = ""
```

Comment:
Ah sorry I think I confused you, the only place I use non-pythonic camelCase variables is QML exposed things, in the data class and really everywhere else it's normal Python convention, i.e. `self.profile_name` for this one here unless you intend to expose it via a property.

Reply:
n/p, I thought *Data class was directly linked to QML, but it's via *Model getter/setters - new to Qt/QML.